pkg/logger/logger.go

```diff
@@ -488,6 +482,20 @@ func Init(cfg *LoggerConfig) {
 	}
 
 	SetBase(NewLogger(cfg))
+
+	// Flush aggreated logs every 3 seconds
+	if pkgLogger.cfg.Aggregate {
+		go func() {
+			for range time.Tick(3 * time.Second) {
```

Comment:
Should this interval be hard coded ? Can't it be passed as an argument through cmdline (together with the cmdline option being added ?). I mean, I might want flushes every 10 seconds, let's say, because I'm debugging something (just thinking o-loud).

Reply:
As commented below: https://github.com/aquasecurity/tracee/pull/2676#discussion_r1097280368